server/bleep/src/analytics.rs

```diff
@@ -123,11 +133,39 @@ impl RudderHub {
         }
     }
 
-    pub fn track_query(&self, user: &crate::webserver::middleware::User, event: QueryEvent) {
+    pub fn identify(&self, self_serve: bool, org_name: Option<&str>, remote_login: &str) {
```

Comment:
We should rename `self_serve` to `cloud`. Also, `remote_login` is a little confusing, since this looks like it's the remote host username - could have `remote_username` instead.